### ISSUE TITLE
Psionic Tassadar Trained Executor Phase 4: implement a real executor transformer family

### ISSUE THREAD

**Issue by AtlantisPleb** [OpenAgentsInc/openagents#3780]
## Summary

Implement a real executor transformer model family for Tassadar in psionic-models, replacing the current fixture-only posture as the train target.

## Why

There is currently no executor model whose forward pass predicts the next trace token. Until that exists, Psionic has substrate but not a trained in-model executor.

## Scope

- add a small executor transformer family in psionic-models
- make its descriptor explicitly executor-specific rather than fixture-only
- carry initial constrained head-geometry claims for the executor path
- add token embeddings, sequence forward pass, and output logits over the executor vocabulary
- keep runtime and model contracts explicit about what is neural execution versus fixture simulation

## Dependencies / Related Issues

- Epic: #3776
- Required prior work: #3779
- Existing substrate dependencies: #3716, #3736, #3713, #3735

## Acceptance Criteria

- Psionic has a trainable executor model family rather than only ProgrammaticFixture bundles
- the model can run a forward pass over tokenized executor sequences and emit logits for next-token training
- descriptor and runtime surfaces remain explicit about geometry, ABI, and decode identity

**Comment by AtlantisPleb** [comment]
Implemented in ce8dde3f9 (`psionic: add Tassadar executor transformer family`). This lands the first real neural executor family in `psionic-models`: a dedicated Tassadar executor transformer descriptor/config, fixed 2D lookup-head geometry claims, next-token forward logits over the tokenized Sudoku-v0 executor vocabulary, linear decode state, and an explicit next-token-only claim boundary so the trained lane stays clearly separated from the exact handcrafted fixture executor.

Verification:
- `cargo test -p psionic-models tassadar -- --nocapture`
